### DISCUSSION TITLE
Try to open codespaces via browser got "Error 503 certificate has expired"

### DISCUSSION THREAD

**Discussion by tempdeltavalue** [community/community#8213] [Codespaces] [answered]
After tap on "get started" button
![Screenshot 2021-11-29 152605](https://user-images.githubusercontent.com/36921178/143868381-955fc039-c097-497a-acd6-313fc27a1953.png)

**Comment by dev** [comment, marked as answer]
GH is already working on this :) 
https://www.githubstatus.com/
